src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2278,16 +2283,15 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
                 {
                     case BoundKind.PropertyAccess:
                     case BoundKind.IndexerAccess:
-                        return CheckIsVariable(argumentSyntax, arg, BindValueKind.RefOrOut, false, diagnostics);
+                        CheckIsVariable(argumentSyntax, arg, BindValueKind.RefOrOut, false, diagnostics);
+                        return;
```

Comment:
Should the return value of `CheckIsVariable` be assigned to `hadError` to match previous behavior?